app/controllers/projects_controller.rb

```diff
@@ -10,6 +10,8 @@ def index
   private
 
   def all_projects
-    Project.includes(:categories).active.joins(:tenant)
+    Project.paginate(page: params[:page],
+                     per_page: 10)
+                    .includes(:categories).active.joins(:tenant)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 16) spaces for indenting an expression spanning multiple lines.